coretex/cli/modules/docker.py

```diff
@@ -0,0 +1,85 @@
+from ...utils import command, CommandException
+
+
+def checkDockerAvailability() -> None:
+    try:
+        # Run the command to check if Docker exists and is available
+        command(["docker", "ps"], ignoreStdout = True)
+    except CommandException as ex:
+        if "not found" in str(ex):
```

Comment:
There is no need to have 3 different messages raised here. We can just do `raise RuntimeError("Docker not available. Please check that it is properly installed and running on your system.")`